srcpkgs/tuimoji/template

```diff
@@ -3,9 +3,10 @@ pkgname=tuimoji
 version=1.0.0
 revision=7
 build_style=python3-module
-pycompile_module="tuimoji"
+#pycompile_module="tuimoji"
```

Comment:
Don't leave template directives commented out. Remove the directive if it is no longer relevant.

Reply:
done